docs/courses/robotics/3.1-pumps-and-pipettes.md

```diff
@@ -119,6 +157,12 @@ set_position(1.9)  # Almost full extension
 pwm.deinit()  # Deinitialize PWM
 ```
 
+#### Cool Uses 😎
+
+- 🍫 Chocolatiers use them for adding precise amounts of flavoring
```

Comment:
Some of these seem a bit ChatGPT-like. Is this something we trust? If not, maybe remove this line